core/src/main/java/org/elasticsearch/action/search/ClearScrollResponse.java

```diff
@@ -69,6 +69,8 @@ public RestStatus status() {
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.field(Fields.SUCCEEDED, succeeded);
+        builder.field(Fields.FOUND, numFreed > 0);
```

Comment:
I think it'd be marginally nicer to return the number freed rather than a boolean here.